### ISSUE TITLE
The game does not run properly when using this mod under Fabric Loader 0.14.23.

### ISSUE THREAD

**Issue by LittleCircleOO** [Gaming32/mod-loading-screen#22]
When I use this mod under Fabric Loader 0.14.23, the game reports an error. Downgrading the Fabric Loader or removing the mod does not cause this problem.
```
Uncaught exception in thread "main"
java.lang.reflect.InvocationTargetException: null
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:?]
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77) ~[?:?]
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:?]
	at java.lang.reflect.Method.invoke(Method.java:568) ~[?:?]
	at oolloo.jlw.Wrapper.invokeMain(Wrapper.java:60) ~[?:?]
	at oolloo.jlw.Wrapper.main(Wrapper.java:53) ~[?:?]
Caused by: java.lang.Error: java.lang.ClassNotFoundException: net.fabricmc.loader.impl.entrypoint.EntrypointUtils
	at io.github.gaming32.modloadingscreen.ModLoadingScreen.<clinit>(ModLoadingScreen.java:200) ~[mod-loading-screen-1.0.3.jar:?]
	at java.lang.Class.forName0(Native Method) ~[?:?]
	at java.lang.Class.forName(Class.java:467) ~[?:?]
	at net.fabricmc.loader.impl.FabricLoaderImpl.setupLanguageAdapters(FabricLoaderImpl.java:470) ~[fabric-loader-0.14.23.jar:?]
	at net.fabricmc.loader.impl.FabricLoaderImpl.finishModLoading(FabricLoaderImpl.java:351) ~[fabric-loader-0.14.23.jar:?]
	at net.fabricmc.loader.impl.FabricLoaderImpl.freeze(FabricLoaderImpl.java:114) ~[fabric-loader-0.14.23.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:147) ~[fabric-loader-0.14.23.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68) ~[fabric-loader-0.14.23.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.14.23.jar:?]
	... 6 more
Caused by: java.lang.ClassNotFoundException: net.fabricmc.loader.impl.entrypoint.EntrypointUtils
	at jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassDelegate.loadClass(KnotClassDelegate.java:226) ~[fabric-loader-0.14.23.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClassLoader.loadClass(KnotClassLoader.java:119) ~[fabric-loader-0.14.23.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:525) ~[?:?]
	at java.lang.Class.forName0(Native Method) ~[?:?]
	at java.lang.Class.forName(Class.java:375) ~[?:?]
	at io.github.gaming32.modloadingscreen.ModLoadingScreen.init(ModLoadingScreen.java:75) ~[mod-loading-screen-1.0.3.jar:?]
	at io.github.gaming32.modloadingscreen.ModLoadingScreen.<clinit>(ModLoadingScreen.java:198) ~[mod-loading-screen-1.0.3.jar:?]
	at java.lang.Class.forName0(Native Method) ~[?:?]
	at java.lang.Class.forName(Class.java:467) ~[?:?]
	at net.fabricmc.loader.impl.FabricLoaderImpl.setupLanguageAdapters(FabricLoaderImpl.java:470) ~[fabric-loader-0.14.23.jar:?]
	at net.fabricmc.loader.impl.FabricLoaderImpl.finishModLoading(FabricLoaderImpl.java:351) ~[fabric-loader-0.14.23.jar:?]
	at net.fabricmc.loader.impl.FabricLoaderImpl.freeze(FabricLoaderImpl.java:114) ~[fabric-loader-0.14.23.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.init(Knot.java:147) ~[fabric-loader-0.14.23.jar:?]
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:68) ~[fabric-loader-0.14.23.jar:?]
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23) ~[fabric-loader-0.14.23.jar:?]
	... 6 more
```

**Comment by Gaming32** [comment]
Fabric 0.14.23 rewrote (or rather, completely moved) core code that the mod relies on, making it entirely incompatible for now. I will be porting soon.

**Comment by Gaming32** [comment]
Fixed in 0db21e07759cc489184c00ddab50388dd17e17da

**Comment by Gaming32** [comment]
Now released in [1.0.4](https://modrinth.com/mod/mod-loading-screen/version/1.0.4).
